src/osd/PG.cc

```diff
@@ -760,7 +756,7 @@ void PG::set_probe_targets(const set<pg_shard_t> &probe_set)
 
 void PG::send_cluster_message(
   int target, MessageRef m,
-  epoch_t epoch, bool share_map_update=false)
+  epoch_t epoch, bool share_map_update)
```

Comment:
why?

Reply:
It's a virtual function, and as such - better not have optional parameters marked by the derivatives.
(I would remove such optional flags whenever possible, any way. They are a relic of the times when we
didn't have overloads in C++, and create more problems than they help)

Reply:
Move this to its own commit.